packages/privacy-scan-runner/src/runner/runner.spec.ts

```diff
@@ -244,10 +246,14 @@ function setupProcessScanResult(): void {
         if ((privacyScanResults.error as BrowserError)?.errorType === 'BannerXPathNotDetected') {
             runState = pageScanResult.run?.retryCount >= maxFailedScanRetryCount ? 'completed' : 'failed';
         }
+        const scanError = privacyScanResults.error as ScanError;
+        if (isEmpty(scanError.errorType)) {
+            scanError.errorType = 'InternalError';
+        }
```

Comment:
Nit (errorType will be overwritten if it exists in privacyScanResults.error)
```suggestion
        const scanError: ScanError = {
            errorType: 'InternalError'
            ...privacyScanResults.error,
        };
```